app/models/claim.rb

```diff
@@ -144,9 +144,9 @@ class Claim < ApplicationRecord
 
   validates :academic_year_before_type_cast, format: {with: AcademicYear::ACADEMIC_YEAR_REGEXP}
 
-  validates :has_student_loan, on: [:"student-loan"], inclusion: {in: [true, false]}
+  validates :has_student_loan, on: [:"student-loan"], inclusion: {in: [true, false]}, allow_nil: true
```

Comment:
probably too much of a faff to change but this should eventually be converted to an `enum` or something if it's representing more than 2 states